Prace_domowe/Praca_domowa3/Wolny_Krzysztof/PD3.ipynb

```diff
@@ -0,0 +1,633 @@
+{
```

Comment:
<p>W jaki sposób to jest mean_acc? Z czego to jest średnia?</p><p> </p><p>Jaki jest powód użycia f1_score z average='micro'?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/208/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>